Docs 7087 Kim Edit (#579)
mulesoft/docs-connectors: modules/ROOT/pages/scripting/scripting-reference.adoc

| authentication     |                       | Information about the authentication.
| flow               | String                | The name of the current flow.
| log                | Logger                | Logger object that logs to the app's log file, `MULE_HOME/logs/<app-name>.log`

> | result             | Object                | Variable representing the result of the script execution.  If the code doesn't produce an implicit result, you have to set this variable explicitly with the execution result. The payload will be set with this value unless you have configured a xref:mule-runtime::target-variables.adoc[Target Variable] as described in xref:scripting/scripting-module.adoc[the scripting module documentation].

Reviewer: I've not yet seen this kind of view in AsciiDoc, with all the space; is it okay?

Reviewer: ```suggestion
| result             | Object                | Variable representing the result of the script execution.  If the code doesn't produce an implicit result, set this variable explicitly with the execution result. The payload is set with this value unless you have configured a xref:mule-runtime::target-variables.adoc[target variable] as described in xref:scripting/scripting-module.adoc[the scripting module documentation].
```
